Palace.xcodeproj/project.pbxproj

```diff
@@ -3920,7 +3936,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 54;
+				CURRENT_PROJECT_VERSION = 55;
```

Comment:
55 already is on TestFlight

Reply:
Ok, i'll merge it anyway. The next PR bumps to 56